webhooklisteners/webhooksharedlist.go

```diff
@@ -0,0 +1,35 @@
+package webhooklisteners
+
+import (
+//"time"
+)
+
+// WebHookSharedListFactory provides the input values to create a
+// WebHookSharedList service object with New()
+type WebHookSharedListFactory struct {
+}
+
+// New creates a new WebHookSharedList service object
+func (w *WebHookSharedListFactory) New() (out *WebHookSharedList, err error) {
+	// TODO do the work
+	return
+}
+
+// WebHookSharedList is the interface that provides the shared list of webhooks
+// across the system
+type WebHookSharedList struct {
+	// TODO add parameters needed to operate the service
```

Comment:
If all you need is a simple slice, consider just having this as `type WebHookSharedList []WebHookListener`